src/odoc/bin/main.ml

```diff
@@ -222,7 +222,7 @@ end = struct
 
   let compile hidden directories resolve_fwd_refs dst output_dir package_opt
       parent_name_opt parent_id_opt open_modules children input warnings_options
-      unique_id =
+      unique_id short_title =
```

Comment:
Perfect for making the experience consistent on every packages. The short title can be removed from `doc/index.mld` when the driver start to use this flag.